### PR TITLE
Update INSTALL.md

### PR DESCRIPTION
**Pull request by Aricg**
Update instructions to compile for online play. (fedora packages have been broken for some time)
Also, add note to update certs for fedora, as Fedora doesn't setup mono correctly.

### CODE REVIEW

**Comment by GraionDilach** [comment]
I don't like what you're doing in the first hunk.

You're pretty much replacing our bleed/nightly aimed-to-developers documentation/readme with a documentation aimed for players(?). I have nothing against the Fedora note in particular but the first hunk doesn't belong to README.md.

**Comment by Mailaender** [comment]
> fedora packages have been broken for some time

Can you make a ticket for that?

File: INSTALL.md [hunk 1]
```diff
@@ -65,6 +68,13 @@ Gentoo
 * gnome-extra/zenity
 * net-misc/curl
 
+Fedora
+=====
+
+```
+sudo cert-sync /etc/pki/tls/certs/ca-bundle.crt
```
Comment (Mailaender) [inline INSTALL.md]: This should be done in the Fedora **mono-core** RPM package in the `%post` section. If it doesn't, you should report at the RedHat Bugzilla.

**Comment by GraionDilach** [comment]
@Mailaender We had an IRC discussion yesterday night that the appimages should fix most of the packaging issues excluding the missing certificates.

**Comment by pchote** [comment]
> I have nothing against the Fedora note in particular but the first hunk doesn't belong to README.md.

Why not both?  IMO it is better for everyone if we can have instructions to compile the development branch *and* the last release version (note: the PR as is will take any tag, so should `| grep "release-"`).

> appimages should fix most of the packaging issues

Not quite.  The appimages *can* fix this, by copying the workaround from https://github.com/flathub/net.openra.OpenRA/blob/master/net.openra.OpenRA.json#L233, but this still needs to be set up (I will file a PR).

**Comment by IceReaper** [comment]
Appimage is not the master solution. It only works if fuse is available, which in case of lxc containers (and im sure other cases too) is not. So additional info is a good idea

**Comment by obrakmann** [comment]
Thank you for your PR, but I believe that this change is not necessary

**Pull request closed without merging**